src/kirby/components/avatar/avatar.component.tns.ts

```diff
@@ -63,5 +66,6 @@ export class AvatarComponent implements OnInit {
       }
     }, 100);
   }
-
 }
+
+registerElement(AVATAR_COMPONENT_SELECTOR, () => require('./avatar.component').AvatarComponent);
```

Comment:
Out of curiosity, this is done to support native code in the component or?

Reply:
It's done to support layouts non @Input properties, in this usecase to use AvatarComponent with the row- and col-attributes used to control layout with GridLayout.